### DISCUSSION TITLE
Make window sticky/ follow groups

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [qtile/qtile#3221] [Q&A]
How to make a window follow me while changing groups? If this is possible i would like to access this feature through the shell, for scripting, like the float argument (`qtile run-cmd **-f** kitty -e tty-clock`).

**Comment by m-col** [comment]
I have good news and bad news. The good news is that you can make windows static using the 'static' command on a window: `qtile cmd-obj -o window -f static`. The bad news is that it has some problems, such as getting it to have keyboard focus, and currently it cannot be made un-static again, so if you want to tile it, well, it needs restarting. For more discussion on improving these windows: https://github.com/qtile/qtile/issues/3243
